ReactQt/runtime/src/netinfo.cpp

```diff
@@ -12,31 +12,111 @@
  */
 
 #include <QNetworkAccessManager>
+#include <QTimer>
+#include <QNetworkRequest>
+#include <QNetworkReply>
+#include <QDebug>
+
+#include <memory>
 
 #include "bridge.h"
 #include "eventdispatcher.h"
 #include "netinfo.h"
 
 namespace {
 static QMap<QNetworkAccessManager::NetworkAccessibility, QString> accessibleName{
-    {QNetworkAccessManager::UnknownAccessibility, "UnknownAccessibility"},
-    {QNetworkAccessManager::NotAccessible, "NotAccessible"},
-    {QNetworkAccessManager::Accessible, "Accessible"}};
+    {QNetworkAccessManager::UnknownAccessibility, "unknown"},
+    {QNetworkAccessManager::NotAccessible, "none"},
+    {QNetworkAccessManager::Accessible, "wifi"}};
 }
 
 class NetInfoPrivate {
 public:
     void monitorNetworkAccess() {
+
+
         QObject::connect(bridge->networkAccessManager(),
                          &QNetworkAccessManager::networkAccessibleChanged,
                          [=](QNetworkAccessManager::NetworkAccessibility accessible) {
                              bridge->eventDispatcher()->sendDeviceEvent(
                                  "networkStatusDidChange",
-                                 QVariantList{QVariantMap{{"network_info", accessibleName.value(accessible)}}});
+                                 QVariantMap{{"connectionType", accessibleName.value(accessible)}});
                          });
+
+        timer = new QTimer();
```

Comment:
Having `timer` on stack should work fine here.

Reply:
Fixed.